### PR TITLE
Added argparse `net` option

### PR DESCRIPTION
**Pull request by JoOkuma**
Hi, first of all, great article, and thanks for making the code publically available.

I noticed that `imageopt` wasn't working with other loss functions. For example:

```bash
$ imageopt --loss BlurredMSELoss --net vgg --invert --seed 1234 --width 300 --lines 2000 --init-sigma2 1.0 --final-sigma2 1.0 --iters 500 --lr 0.01 --init-raster results/vancouver/init.png --final-raster results/vancouver/final.png --init-pdf results/vancouver/init.pdf --final-pdf results/vancouver/final.pdf --snapshots-path results/vancouver data/vancouver.jpg --snapshots-steps 100 --colour

imageopt: error: unrecognized arguments: --net data/vancouver.jpg
```

The `net` parameter was missing and this PR fix this issue.

### CODE REVIEW

**Comment by jonhare** [comment]
Closing because adding a `--net` option to `(Blurred)MSELoss` and `BCELoss` doesn't make sense - they are static functions and do not use a neural network to calculate the loss.
